src/services/member/plugins/profile/service.ts

```diff
@@ -1,28 +1,24 @@
 import { FastifyBaseLogger } from 'fastify';
 
-import { UnauthorizedMember } from '../../../../utils/errors';
+import { UUID } from '@graasp/sdk';
+
 import { Repositories } from '../../../../utils/repositories';
-import { Actor } from '../../../member/entities/member';
+import { Actor, Member } from '../../../member/entities/member';
 import { IMemberProfile } from './types';
 
 export class MemberProfileService {
-  log: FastifyBaseLogger;
+  private log: FastifyBaseLogger;
 
   constructor(log: FastifyBaseLogger) {
     this.log = log;
   }
 
-  async post(member: Actor, repositories: Repositories, data: IMemberProfile) {
-    const { memberProfileRepository } = repositories;
-    if (!member?.id) {
-      throw new UnauthorizedMember();
-    }
+  async post(member: Member, { memberProfileRepository }: Repositories, data: IMemberProfile) {
     const profile = await memberProfileRepository.createOne({ ...data, member });
     return profile;
   }
 
-  async get(memberId: string, repositories: Repositories) {
-    const { memberProfileRepository } = repositories;
+  async get(_actor: Actor, { memberProfileRepository }: Repositories, memberId: UUID) {
```

Comment:
Since we're not using the `_actor` parameter, should we simply remove it from the function definition ?